src/packaging/src-dist/MANIFEST.yaml

```diff
@@ -191,6 +191,7 @@ tarballs:
             - mutex.h
             - random.h
             - types.h
+            - unaligned.h
```

Comment:
MANIFEST changes depend on what we do per my previous comment. But I really do not want to be including both files in the manifest.